code/modules/fishing/fishing_spots.dm

```diff
@@ -524,14 +547,17 @@ ABSTRACT_TYPE(/datum/fishing_spot)
 
 /datum/fishing_spot/lava_moon
 	fishing_atom_type = /turf/unsimulated/floor/lava
-	rod_tier_required = 3
-	fish_available = list(/obj/item/reagent_containers/food/fish/lava_fish = 25,\
-	/obj/item/reagent_containers/food/fish/igneous_fish = 10,\
+	rod_tier_required = 2
+	fish_available = list(/obj/item/reagent_containers/food/fish/igneous_fish = 10,\
```

Comment:
Same as with the clown shoes, if we're doing loot tables, let's try to clean up all of the individual assignments. Not saying do this for every fishing spot in this PR, but if you start messing with balance (like lowering rod tier) then you should do it proper.